RELEASE.md

```diff
@@ -1,3 +1,40 @@
+# Release 2.2.0
+
+The 2.2 minor series tracks TensorFlow 2.2.
+
+## Features
+
+- Profile plugin now should be pip installed.
+- Debugger:
+  - Added support for alerts (#3285)
+  - Support FULL_HEALTH debug mode (#3343)
+  - Integration with ngrx storage (#3359)
```

Comment:
Are these actually user-facing features yet? My understanding was that
the new UI is only available in “ng-tensorboard” and so isn’t actually
included in the Pip package. (Also, we try to keep the release notes
more user-understandable than “Integration with ngrx storage”, which
doesn’t really communicate why users would want this.)

Reply:
Removed.